src/applications/disability-benefits/all-claims/content/disabilityLabelsRevised.js

```diff
@@ -11,523 +11,572 @@ export default {
   7: 'ALS (amyotrophic lateral sclerosis)',
   8: 'anal cancer',
   9: 'anal or perianal fistula (fistula-in-ano)',
-  10: 'anemia',
```

Comment:
just noting `anemia` seems to be cut entirely. accurate?

Reply:
That term was removed from the autosuggestion list for this release so that is accurate.